### PR TITLE
Add participant

### PR DESCRIPTION
**Pull request by maticzav**
#1 referenced!

### CODE REVIEW

File: PARTICIPANTS/maticzav.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Matic Zavadlal
```
Comment (MJ10) [inline PARTICIPANTS/maticzav.txt]: The name of the directory should be `Participants`

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

**Comment by maticzav** [comment]
Hi! No, I shall fix this today. Hope that you can wait :smile:

**Comment by maticzav** [comment]
Fixed changes in #467 as I was unable to change directory name from `PARTICIPANTS` to `Participants` in local file system. Hope I fixed everything and thank you so much that you waited.

**Pull request closed without merging**